src/ControlSystem/Measurements/BNSCenterOfMass.hpp

```diff
@@ -14,6 +14,10 @@
 #include "DataStructures/LinkedMessageId.hpp"
 #include "DataStructures/Tensor/TypeAliases.hpp"
 #include "Domain/Structure/ObjectLabel.hpp"
+#include "Evolution/DgSubcell/ActiveGrid.hpp"
+#include "Evolution/DgSubcell/Tags/ActiveGrid.hpp"
+#include "Evolution/DgSubcell/Tags/Coordinates.hpp"
+#include "Evolution/DgSubcell/Tags/Mesh.hpp"
```

Comment:
This will depend the ControlSystem library, and hence also BBH simulations, on Subcell. I think we have to find a way around that. Maybe base tags or something like that, as @knelli2 suggested? @nilsdeppe any thoughts?

Reply:
Use the `EventsAndTriggers` tags and an observation box? That's the easiest way to decouple consumption from addition with DataBox